mpi-proxy-split/util/mpi-logger/README

```diff
@@ -0,0 +1,17 @@
+## MPI-LOGGER
+The purpose of mpi-logger is to intercept MPI calls from native program without involving running MANA.
+Using this tool people can compare MPI arguments between MANA and native run, in order to debug MANA.
+
+To build, go to ./mpi-proxy-split/util/mpi-logger and run `make`
+
+## How to run
+To run on Perlmutter, preload the generated .so file before you run.
```

Comment:
Suggested re-wording:
> To run, use LD_PRELOAD to preload the generated .so file before you run.

(I'm removing `Perlmutter`, since this code will work on most computers.)